libraries/AP_GPS/AP_GPS_UBLOX.h

```diff
@@ -73,6 +73,7 @@
 #define CONFIG_SBAS          (1<<12)
 #define CONFIG_RATE_PVT      (1<<13)
 #define CONFIG_TP5           (1<<14)
```

Comment:
Is that really a `5`?!

Reply:
Yes. (The ublox message is called UBX-CFG-TP5)